main/install/configuration.dist.php

```diff
@@ -1967,3 +1967,9 @@
 // You need add a new option called "confirmation" to the registration settings
 //INSERT INTO settings_options (variable, value, display_text) VALUES ('allow_registration', 'confirmation', 'MailConfirmation');
 // ------ (End) Custom DB changes
+
+// Configuration setting to disable course code field in course creation form.
+//$_configuration['course_creation_form_hide_course_code'] = false;
+
+// Configuration setting to make required course category in course creation form.
+//$_configuration['course_creation_form_set_course_category_mandatory'] = false;
```

Comment:
Agregar las nuevas variables antes de la línea 1963 que indica "keep this at the end"